### PR TITLE
Translation into Russian, adding translation and initial work

### PR DESCRIPTION
**Pull request by aksenovaa**

### CODE REVIEW

File: Documentation/Localization.ru_RU/About.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
+.. include:: /Includes.rst.txt
+
+
+.. _introduction:
+
+.. _about:
+
+================
+Об этом руководстве
+================
```
Comment (linawolf) [inline Documentation/Localization.ru_RU/About.rst:10]: These lines must always be as long or longer then the text

**Comment by linawolf** [comment]
@brotkrueml please have a look as my russian is even more limited then yours

File: Documentation/Localization.ru_RU/About.rst (hunk 1 above)
Comment (aksenovaa) [reply Documentation/Localization.ru_RU/About.rst:10]: Done in the latest update

**Review by linawolf — APPROVED** [review]

**Comment by linawolf** [comment]
Let us merge it like this, we can always do more fixes in a follow up

**Comment by aksenovaa** [comment]
Yeah, well, I'll translate little by little, in batches. Like a chapter at a time.

**Comment by linawolf** [comment]
Thanks for the great work

**Pull request merged**